handbook/engineering/leadership/index.md

```diff
@@ -0,0 +1,80 @@
+- Plan: We are defining a new fullstack role and interview process to help us find engineers who can do the necessary frontend work, but also contribute to our backend systems. We think this role could be valuable for other teams, like Search and Campaigns, in the future. 
+- Owners: Tomás, Aida
+- Status: [In progress](https://github.com/sourcegraph/interviews/pull/136).
+
+#### Engineering manager onboarding
+
+- Problem: We recently hired and onboarded our first group of engineering managers and onboarded each of them in an ad hoc way. We will be hiring at least one more engineering manager in the near future (for the web team) and we want to document what we have learned so they have an even better onboarding experience.
+- Owner: Nick
+- Plan: Document the onboarding process for engineering managers in our handbook before the EM for the web team starts.
+- Status: Not started.
+
+### Medium term (~3-6 months)
+
+#### Publicly documented compensation bands
+
+- Problem: When we were smaller, it was easy to contain consistent compensation across the entire team because there were fewer people to coordinate with. Now that we are bigger and have more engineering managers making offers and compensation decisions, we need to have clearly documented compensation bands so we can maintain consistency while federating decision making. We want to make this public because it will help candidates understand what they can expect before they even apply. This is a better candidate experience and it avoid rare but real situations where a candidate goes through our interview process only to discover that their compensation expectations don't fit into our bands.
+- Owner: Nick
+- Plan:
+  1. Update our benchmark data and set bands to be shared with engineering managers internally to help them make offers and review/adjust compensation for their teams.
+  2. Update the [compensation documentation in our handbook](../../people-ops/compensation.md#ranges) so candidates and current teammates know what our bands are.
+- Status: [First milestone is in progress](https://docs.google.com/spreadsheets/d/1iQsfchsnY5CxKSZfcnazpnJ1uSL6-gMuEI-wgsfLq3I/edit#gid=1795648823).
+
+#### Publicly documented interview modules
+
+- Problem: Any candidate who has interviewed at multiple companies knows that interview processes are hit or miss. We have spent time and effort to ensure that we have a great interview process, and candidates regularly tell us that we do after going through it. Right now, candidates don't know what to expect out of our interview process beyond the title of each interview module and a brief description.
+- Plan: We want candidates to know up front as much as possible about each interview, including exact questions that we will be asking them as and how we will be evaluating their answers (as much as possible). Sharing this information would have many benefits:
```

Comment:
s/them as/them/

Reply:
Thanks! Fixed in 35c018d22209f2625d23b8ec330ad56699b66352